app/human-detector-app/classes/Camera.tsx

```diff
@@ -0,0 +1,55 @@
+import Group from './Group';
+
+export default class Camera {
+  //field
+  cameraId:string;
```

Comment:
ID should be a UUID (see [this file](https://github.com/tucker-moore/human-detector/blob/fd3081ff3adf430a6e0569dc6821a4d77401280a/server/backend/src/cameras/camera.entity.ts#L12) and the [database diagram](https://docs.google.com/document/d/1kcKgzWn7GJJ9U0C9lfVgtobQYh-9a3MP--C6RZD7CGs/edit#heading=h.kgmd1an91ao5))

Reply:
Are you talking about changing the variable name here?  To like `uuid` or `cameraUUID` or something like that?

Reply:
I meant changing `cameraId`'s type to a UUID, but it looks likeNode's `uuid` package stores them as strings anyways so this is fine

Reply:
Ahhhjhh. What is the `v4()` function supposed to be doing in that file you sent?

Reply:
`v4()` just generates a random version 4 UUID. Docs for the package it's from are [here](https://www.npmjs.com/package/uuid) if you wanna read more

Reply:
ahhh i think i get it.  I don't think it's that important to change the type to a uuid no?  isn't that what the app would be sending/receiving